gatewayconfig/processors/led_processor.py

```diff
@@ -14,17 +15,18 @@ def __init__(self, status_led: LED, shared_state: GatewayconfigSharedState):
         self.shared_state = shared_state
 
     def run(self):
-        logger.debug("LED LEDProcessor")
+        LOGGER.debug("LED LEDProcessor")
 
-        while True:
-            # Blink fast if diagnostics are not OK
-            if(self.shared_state.are_diagnostics_ok is False):
-                # logger.debug("Diagnostics are not OK, blinking fast")
-                self.status_led.blink(0.1, 0.1, 10, False)
-            # Blink slow if advertising bluetooth
-            elif(self.shared_state.is_advertising_bluetooth is True):
-                self.status_led.blink(1, 1, 1, False)
-            # Solid if diagnostics are OK and not advertising
-            else:
-                self.status_led.on()
-            sleep(LED_REFRESH_SECONDS)
\ No newline at end of file
+        if is_raspberry_pi():
+            while True:
+                # Blink fast if diagnostics are not OK
+                if(self.shared_state.are_diagnostics_ok is False):
+                    # LOGGER.debug("Diagnostics are not OK, blinking fast")
```

Comment:
Let's not leave commented out code behind. If we use the appropriate log level, this should just be ignored.

Reply:
Decided to remove it all together. This is really noisy when enabled so I don't want it in DEBUG. There's [no TRACE level](https://docs.python.org/3/library/logging.html#levels) in Python's logging.